README_PROCESS.md

```diff
@@ -0,0 +1,221 @@
+## Repository branch structure
+
+The SYCL specification intends to have periodic major versions, where each is
+named according to the year in which it is approved by the working group.
+Although there were previous major versions, this repository starts with the
+SYCL 2020 specification.
+From this point forward, the SYCL specification is "unified", meaning that a
+single document serves as the official specification for all major versions
+from SYCL 2020 onwards.
+
+There are often two active branches in the repository: a branch named "main" and
+a branch whose name has the form "sycl-xxxx" (e.g. "sycl-2020").
```

Comment:
Probably not worth it at this point, but it might be worth exploring if a name like "develop" would help people to understand that the branch does not correspond to a released specification.